package.json

```diff
@@ -66,15 +66,15 @@
     "@rollup/plugin-babel": "^5.2.1",
     "@rollup/plugin-commonjs": "^15.0.0",
     "@rollup/plugin-node-resolve": "^9.0.0",
-    "@storybook/addon-a11y": "5.3.21",
-    "@storybook/addon-actions": "5.3.21",
-    "@storybook/addon-centered": "5.3.21",
```

Comment:
Removes `@storybook/addon-centered`, now deprecated, as new core Storybook settings suffice, as adds `@storybook/theming` explicitly, since we use it in `.storybook/theme.ts`.